cmd/zed/manage/command.go

```diff
@@ -8,8 +8,8 @@ import (
 
 	"github.com/brimdata/zed/cli/lakeflags"
 	"github.com/brimdata/zed/cli/logflags"
-	"github.com/brimdata/zed/cmd/zed/manage/lakemanage"
 	"github.com/brimdata/zed/cmd/zed/root"
+	lakemanage "github.com/brimdata/zed/lake/api/manage"
```

Comment:
I think we should put this in cmd/zed/internal/manage (or maybe cmd/zed/internal/lakemanage).

While I understand the thinking behind putting it here (i.e., that it interacts with lakes only through the lake/api package), everything else in or under the lake package is part of the lake implementation or API. This, on the other hand, is an API consumer. Moreover, it's intended for use only under cmd/zed.

Reply:
Not opposed to doing this but I think if you're going to import the package into `service` as we do in this pr, its a little weird to have inside of `cmd/zed`.

Reply:
But this PR doesn't touch package service.

Reply:
oh wait you are right. Ok we can try this.